Sources/Quick/QuickSpec.m

```diff
@@ -115,10 +115,23 @@ + (SEL)addInstanceMethodForExample:(Example *)example {
     }
 
     const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(id), @encode(id), @encode(SEL)] UTF8String];
-    NSString *selectorName = [NSString stringWithFormat:@"%@_%@_%ld",
-                              example.name.qck_selectorName,
-                              sanitizedFileName,
-                              (long)example.callsite.line];
+    
+    NSString *originalName = example.name.qck_selectorName;
+    NSString *selectorName = originalName;
+    unsigned int i = 2;
```

Comment:
If you're referring to warnings when converting `NSUInteger` to `%lu`, I think the best solution is using `%tu` (for `NSUInteger`) or `%zu` (for `NSInteger`). That way you can use string formatting without a cast! :+1: